util/geometry/src/test/java/jetbrains/jetpad/geometry/DoubleRectangleEdgesTest.java

```diff
@@ -0,0 +1,50 @@
+package jetbrains.jetpad.geometry;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class DoubleRectangleEdgesTest {
+
+  private static final double LEFT = 5.;
+  private static final double RIGHT = 15.;
+
+  private static final double TOP = 40.;
+  private static final double BOTTOM = 60.;
+
+  private static final DoubleVector LEFT_TOP = new DoubleVector(LEFT, TOP);
+  private static final DoubleVector LEFT_BOTTOM = new DoubleVector(LEFT, BOTTOM);
+  private static final DoubleVector RIGHT_TOP = new DoubleVector(RIGHT, TOP);
+  private static final DoubleVector RIGHT_BOTTOM = new DoubleVector(RIGHT, BOTTOM);
+
+  private static final DoubleSegment LEFT_EDGE = edge(LEFT_TOP, LEFT_BOTTOM);
+  private static final DoubleSegment BOTTOM_EDGE = edge(LEFT_BOTTOM, RIGHT_BOTTOM);
+  private static final DoubleSegment RIGHT_EDGE = edge(RIGHT_TOP, RIGHT_BOTTOM);
+  private static final DoubleSegment TOP_EDGE = edge(LEFT_TOP, RIGHT_TOP);
+
+  private static final DoubleRectangle RECTANGLE = new DoubleRectangle(LEFT_TOP, RIGHT_BOTTOM.subtract(LEFT_TOP));
+
+  @Test
+  public void checkLeftEdge() throws Exception {
```

Comment:
Check is a bad prefix for a test. Also why do you add throws Exception?

Reply:
I'll name them same as other simple tests - just method name, because there are no special conditions in these tests. Throws were added automatically when IDEA generated test methods and I missed them. I changed template to avoid such issues in the future.